Lib9c/Action/RequiredAppraiseBlockException.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    public class RequiredAppraiseBlockException : Exception
```

Comment:
Can you add a summary for this exception? (because I couldn't figure out what kind of error this was.)